### ISSUE TITLE
Importing SELECT causes error

### ISSUE THREAD

**Issue by tslater** [Semantic-Org/Semantic-UI-React#368]
I'm using es6 and webpack. 
When I import `Select` like so:

```
import {Select} from 'stardust';
```

I get this error:

```
warning.js:44Warning: Invalid argument supplied to oneOf, expected an instance of array.
```

It seems to be on line 133 of Label.js:

```
  /** Color of the label. */
  color: _react.PropTypes.oneOf(Label._meta.props.colors),
```

**Comment by tslater** [comment]
Filter.js is my own class where I import SELECT

![screen shot 2016-08-03 at 5 57 25 pm](https://cloud.githubusercontent.com/assets/449326/17386021/d0b1afd4-59a3-11e6-9e6b-c2fa7ebdba0d.png)

**Comment by levithomason** [comment]
Yep, that is a typo bug. PR and release coming, hold tight.

**Comment by levithomason** [comment]
Though, this warning shouldn't stop the app from running.  It should only be logged as an error by React propTypes.

**Comment by levithomason** [comment]
Fixed and released in `v0.27.6`, thanks for the detailed report.

**Comment by tslater** [comment]
No problem. That was a fast response! I did fail to mention that things were working just fine...sorry about that!
